config/settings.yml

```diff
@@ -293,7 +293,7 @@ modules_appeals_api:
       # Absolute path to the private key file matching the jwk:
       private_key_path: ''
   slack:
-    api_token: ''
+    api_key: ''
```

Comment:
While only tangentially related to the other changes here, I noticed while investigating the `VAForms` job that we had the wrong key names in this file, and so I went ahead and fixed them.